### PR TITLE
[README] Goal updates

### PR DESCRIPTION
**Pull request by jhersh**
I've aimed for simpler language, directness, and more active verbs. I split the last goal in two, the latter half now in the third spot. Some of these are nontrivial! Thoughts welcome.

### CODE REVIEW

**Comment by orta** [comment]
Giphy

I prefer this version, nice work @jhersh

File: README.md [hunk 1]
```diff
@@ -19,29 +19,26 @@ guide](http://guides.cocoapods.org/using/getting-started.html#installation) and
...
-  management systems. (Currently supported are git, svn, mercurial, bazaar, and
-  various types of archives downloaded over HTTP.)
-- Provide the ability to work in a distributed way, but also provide
+CocoaPods aims to improve the engagement with, and discoverability 
+of, third party open-source Cocoa libraries. These
+project goals influence and drive the design of CocoaPods:
+
+- Create and share libraries, and use them in your own projects,
+  without creating extra work for library authors. Hack on your own fork 
+  of any library with a simple transparent `Podspec` standard.
```
Comment (alloy) [inline README.md]: This is now missing an entry about not needing a fork at all in order to be able to provide a podspec, which was more what the original goal tried to describe.

**Comment by alloy** [comment]
@jhersh Thanks for taking the time to improve it!

**Comment by jhersh** [comment]
Thanks @orta @alloy! I updated the first goal to be clearer about integrating non-CocoaPods libraries.

**Pull request merged**